public/js/findMovies.js

```diff
@@ -37,7 +41,9 @@ async function yearFormHandler(event) {
 }
 
 async function genreFormHandler(event) {
-    if (movieGenre) {
+    event.preventDefault();
+
+    if (genre) {
         const response = await fetch('/api/movies/genre', {
             method: 'get',
             body: JSON.stringify({
```

Comment:
This whole function can probably get removed since we won't have genre information.